src/Core/Framework/Script/Exception/HookInjectionException.php

```diff
@@ -5,6 +5,9 @@
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\Framework\Script\Execution\Hook;
 
+/**
+ * @deprecated v6.7.0 - Will be removed in v6.8.0. Use Shopware\Core\Checkout\Cart\CartException::hookInjectionException instead
```

Comment:
this is still used here `\Shopware\Core\Framework\DataAbstractionLayer\Facade\SalesChannelRepositoryFacadeHookFactory::factory` we should also be adjusted then